File: 2714-left-and-right-sum-differences/2714-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int[] answer = new int[nums.length];

        int leftSum = 0;
        int rightSum = 0;
        for(int num: nums) {
            rightSum += num;
        }
        for(int i = 0; i < nums.length; i++) {
            rightSum = rightSum - nums[i];
            answer[i] = Math.abs(rightSum - leftSum);
            leftSum = leftSum + nums[i];
        }
        return answer;
    }
}

File: 2714-left-and-right-sum-differences/2714-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        leftSum = 0
        rightSum = sum(nums)
        answer = [0]*len(nums)
        for i in range(0,len(nums)):
            rightSum = rightSum - nums[i]
            answer[i] = abs(rightSum - leftSum)
            leftSum = leftSum + nums[i]
        
        return answer